frontend/app/utils/locale-utils.ts

```diff
@@ -7,12 +7,13 @@ import { type UNSAFE_RouteModules as RouteModules } from '@remix-run/react';
  */
 export function getLocale(url: string) {
   const pathname = new URL(url).pathname;
+  const locale = pathname.split("/")?.[1]
 
-  if (pathname.startsWith('/en')) {
+  if (locale==="en") {
```

Comment:
Nitpicking: add spaces before and after `===`. Ensure you have prettier plugin installed and confire your editor to auto-format on `.ts` and `.tsx`.

Reply:
Auto-formatters are satan's spawn